tests/integration/standard/test_tablets_routing_v2.py

```
@@ -0,0 +1,545 @@
+from tests.integration import PROTOCOL_VERSION, use_cluster
+
+
+def setup_module(module):
+    try:
+        use_cluster('tablets_routing_v2', [3], start=True, set_keyspace=False,
+                    configuration_options={
+                        # `strongly-consistent-tables` is what gates the server's
+                        # advertisement of TABLETS_ROUTING_V2_EXPERIMENTAL.
+                        'experimental_features': ['lwt', 'udf', 'strongly-consistent-tables'],
+                    })
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)